lib/political_content_identifier.rb

```diff
@@ -0,0 +1,36 @@
+class PoliticalContentIdentifier
+  POLITICAL_PUBLICATION_TYPES = [
+    PublicationType::CorporateReport,
+    PublicationType::ImpactAssessment,
+    PublicationType::InternationalTreaty,
+    PublicationType::PolicyPaper,
+    PublicationType::ResearchAndAnalysis,
+  ].freeze
+
+  def self.political?(edition)
+    has_minister?(edition) || (is_political_format?(edition) && has_political_org?(edition))
+  end
+
+private
+
+  def self.has_minister?(edition)
+    (edition.can_be_associated_with_ministers? && edition.ministerial_roles.any?) ||
+      edition.try(:role_appointment).try(:role).is_a?(MinisterialRole)
```

Comment:
I wonder if this logic belongs in edition rather than here? i.e. an `edition.is_associated_with_a_minister?` method? It's currently encapsulating a lot of knowledge about how different editions and their module mixins behave. Something like this:

```ruby

class Edition
  def is_associated_with_a_minister?
    false
  end
end

module Edition::Ministers
  def is_associated_with_a_minister?
    ministerial_roles.any?
  end
end

module Edition::Appointments
  def is_associated_with_a_minister?
    role_appointment && role.is_a?(MinisterialRole)
  end
end
```